Dockerfile.dapper

```diff
@@ -1,13 +1,13 @@
-FROM quay.io/skopeo/stable:v1.1.1
+FROM quay.io/skopeo/stable:v1.13.2
 
-# Add jq
-RUN yum -y update && yum -y install jq && yum -y clean all && rm -rf /var/cache/dnf/* /var/log/dnf* /var/log/yum*
+# Add jq, git, helm
+RUN yum -y update && yum -y install jq git helm && yum -y clean all && rm -rf /var/cache/dnf/* /var/log/dnf* /var/log/yum*
 
 # Add docker cli
 COPY --from=docker.io/library/docker:19.03.12 /usr/local/bin/docker /usr/local/bin/
 
 # Add buildx plugin from github
-RUN mkdir -p /root/.docker/cli-plugins/ && curl -sLo /root/.docker/cli-plugins/docker-buildx https://github.com/docker/buildx/releases/download/v0.4.2/buildx-v0.4.2.linux-amd64 && chmod a+x /root/.docker/cli-plugins/*
+RUN mkdir -p /usr/libexec/docker/cli-plugins/ && curl -sLo /usr/libexec/docker/cli-plugins/docker-buildx https://github.com/docker/buildx/releases/download/v0.4.2/buildx-v0.4.2.linux-amd64 && chmod a+x /usr/libexec/docker/cli-plugins/*
```

Comment:
Considered using a newer version of `buildx`? It seems that version `0.4` is already too old.

Reply:
This line is from upstream: https://github.com/rancher/image-mirror/blob/8e0b742325dac0ba9fa8653f36c1dea004780eee/Dockerfile.dapper#L10